openedx/core/djangoapps/user_api/preferences/tests/test_api.py

```diff
@@ -181,6 +181,34 @@ def test_update_user_preferences(self):
             "new_value"
         )
 
+    def test_update_user_preferences_with_username(self):
```

Comment:
That is not true for staff users.  Staff users do have access to other user's preferences.  
Are there any success tests for passing a different user object by staff users?